### PR TITLE
Hacky work around for fixing #4732 (resize problem in metadata converter).

### PR DESCRIPTION
**Pull request by ismailsunni**
### What does it fix?
* Ticket: #4732 and #4746
* Funded by: DFAT
* Description: 
  - Hacky work around for fixing #4732.
This is the best that I can do:
metadata_converter_resize

Please check in your computer @Charlotte-Morgan 

### Checklist:
- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [x] Request someone to review or test your PR

### CODE REVIEW

File: safe/gui/tools/metadata_converter_dialog.py [hunk 1]
```diff
@@ -318,3 +322,23 @@ def show_current_metadata(self):
...
+        """
+        self.resized.emit()
+        return super(MetadataConverterDialog, self).resizeEvent(event)
+
+    def after_resize(self):
+        """Method after resizing the window."""
+        # TODO(IS) this is a QT bug :
+        # https://stackoverflow.com/q/25644026/1198772
+        # It's fixed in QT 5.4 https://bugreports.qt.io/browse/QTBUG-37673
+        # If you port it to QGIS 3 (which use QT 5.9, please remove this
```
Comment (Gustry) [inline safe/gui/tools/metadata_converter_dialog.py]: QGIS Minimum QT version is 5.2 https://github.com/qgis/QGIS/blob/master/CMakeLists.txt#L277

**Comment by Charlotte-Morgan** [comment]
Hi @ismailsunni - it looks maybe better than you think.  The window is smaller, roughly in the centre of my screen and all is visible.  I can not change the size, but I can change the location of the window.
LGTM

**Comment by ismailsunni** [comment]
>  I can not change the size

Really? I will check in my windows machine (while waiting Travis to finish)

**Comment by meysita** [comment]
Thanks mas @ismailsunni
sudah bisa sekarang

**Comment by ismailsunni** [comment]
@meysita eh, sudah bisa? Kan belum dirilis patch buat benerin isunya mbak. Atau nge-pull dari github langsung?

**Comment by meysita** [comment]
uda bisa mas @ismailsunni , ini baru aja aku coba di komputer. Awalnya qgis ku masih versi lama trus aku install yang 2.18 trus inasafe juga masih versi lama, tak instal yang versi 4.2.5
Trus uda bisa ini. Kirain uda dibenerin mas

**Comment by ismailsunni** [comment]
Eh, ajaib kalau gitu mbak @meysita :D

**Comment by meysita** [comment]
iya mas @ismailsunni , uda bisa beneran ini

**Comment by meysita** [comment]
image

tapi ini aku pake komputer yang berbeda dengan yang kemaren

**Pull request merged**